src/pyaro/timeseries/AutoFilterReaderEngine.py

```diff
@@ -34,7 +34,7 @@ def supported_filters(cls) -> [Filter]:
         filts = []
```

Comment:
The type annotation of this function is wrong.
Also, it would be cleaner as a list comprehension, as follows:
``` python
    @classmethod
    def supported_filters(cls) -> list[Filter]:
        """Get the default list of implemented filters.
        :return: list of filters
        """
        names = "variables stations countries bounding_boxes duplicates time_bounds flags time_variable_station".split()
        return [filters.get(name) for name in names]
```

Reply:
Though I often find list-comprehensions hard to read, I agree, that it is much cleaner in this case.